pkg/windows/windows.go

```diff
@@ -140,6 +145,11 @@ func parseNetUseOutput(text string) (remoteDrives, error) {
 			continue
 		}
 
+		status := strings.TrimSpace(line[:cols.Local.Start])
+		if status != "" && strings.ToLower(status) != "ok" {
```

Comment:
Should we still parse network drives which aren't `ok`? What if a drive is showing temporarily as unavailable, we would still want to detect the path as a remote drive.

Reply:
the `unavailable` status is often shown when the mapped network drive has an invalid ip address. I demonstrate that in the example.

Reply:
It's ok to still parse though, because the `windowsNetworkMountRgx` should prevent it from being used? I think it's better to parse all listed remote drives in case we have unexpected cases where a valid drive isn't listed as `ok`.

Reply:
Valid drives are never listed as `ok` if them aren't `ok` and accessible. What I did is to consider empty and `ok` statuses to avoid unavailable ones.

Reply:
Would it cause problems to parse it anyway?

Reply:
It doesn't cause problems when parsing but we're taking the risk of using unavailable mappings as valid ones.

Reply:
Can there be duplicate entries for the same drive letter? If not, then it shouldn't matter if we parse unavailable ones because they just won't be used since the entity file path would only be using drives that are readable in the IDE.